pkg/config/app/kuma-dp/config.go

```diff
@@ -189,6 +189,8 @@ type DataplaneRuntime struct {
 	// Available values are: [trace][debug][info][warning|warn][error][critical][off]
 	// By default it inherits Kuma DP logging level.
 	EnvoyLogLevel string `json:"envoyLogLevel,omitempty" envconfig:"kuma_dataplane_runtime_envoy_log_level"`
+	// EnvoyComponentLogLevel configures Envoy's --component-log-level
```

Comment:
Can you add a bit more content re:format?

Reply:
Should it just be "look at the Envoy docs"? This will always correspond to the Envoy version in use, not Kuma.

Reply:
@lahabana I've added a sentence linking to the envoy docs